script.js

```diff
@@ -30,3 +34,149 @@ closingBtn.addEventListener('click', ()=>{
+    technology: ['html', 'css', 'javascript'],
+    liveLink: 'https://kidd254.github.io/Lawrence-s-Portfolio/',
+    sourceLink: 'https://github.com/Kidd254/Lawrence-s-Portfolio',
+    image: './images/Snapshoot Portfolio-pop-mobile.png',
+    image2: './images/Snapshoot Portfolio-pop-desk.png',
+    btnImg: 'images/Vector github.png',
+    btnImg2: 'images/Icon.png',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    list: ['Canopy', 'Back end Dev', '2015'],
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'css', 'javascript'],
+    liveLink: 'https://kidd254.github.io/Lawrence-s-Portfolio/',
+    sourceLink: 'https://github.com/Kidd254/Lawrence-s-Portfolio',
+    image: './images/Snapshoot Portfolio-pop-mobile.png',
+    image2: './images/Snapshoot Portfolio-pop-desk.png',
+    btnImg: 'images/Vector github.png',
+    btnImg2: 'images/Icon.png',
+  },
+  {
+    title: 'Tonic',
+    list: ['Canopy', 'Back end Dev', '2015'],
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'css', 'javascript'],
+    liveLink: 'https://kidd254.github.io/Lawrence-s-Portfolio/',
+    sourceLink: 'https://github.com/Kidd254/Lawrence-s-Portfolio',
+    image: './images/Snapshoot Portfolio-pop-mobile.png',
+    image2: './images/Snapshoot Portfolio-pop-desk.png',
+    btnImg: 'images/Vector github.png',
+    btnImg2: 'images/Icon.png',
+  },
+  {
+    title: 'Multi-Post Stories',
+    list: ['Canopy', 'Back end Dev', '2015'],
+    description:
+        'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard',
+    technology: ['html', 'css', 'javascript'],
+    liveLink: 'https://kidd254.github.io/Lawrence-s-Portfolio/',
+    sourceLink: 'https://github.com/Kidd254/Lawrence-s-Portfolio',
+    image: './images/Snapshoot Portfolio-section1.png',
+    image2: './images/Snapshoot Portfolio-pop-desk.png',
+    btnImg: 'images/Vector github.png',
+    btnImg2: 'images/Icon.png',
+  },
+];
+
+const popUp = (info) => {
+  const popUpContainer = document.createElement('section');
+  popUpContainer.id = 'pop-up-main';
+  popUpContainer.innerHTML = `<div class="pop-content">
+    <div class="pop-up-header">
+      <h1 class="section-titled">${info.title}</h1>
+     
+        <i class="fa-solid fa-xmark" id="btnd"></i>
+     
+    </div>
+    <ul class="dynamic-list">
+      <li class="info1">${info.list[0]}</li>
+      <li class="info2">${info.list[1]}</li>
+      <li class="info3">${info.list[2]}</li>
+      </ul>
+    <div class="pop-up-image"><img src="${info.image}"></div>
+    <div class="pop-up-image-desk"><img src="${info.image2}"></div>
+    <div class="pop-up-content">
+      <p class="pop-up-description">${info.description}</p>
+      </p>
```

Comment:
- [ ] The description of the project in the popup is not the same as the page. You should load in the popup the same data coming to the page

<img width="1439" alt="Screenshot 2023-01-07 at 11 18 27 AM" src="https://user-images.githubusercontent.com/40503704/211146542-59cdee07-796c-4993-86c4-8270344d3552.png">
<img width="703" alt="Screenshot 2023-01-07 at 11 18 45 AM" src="https://user-images.githubusercontent.com/40503704/211146544-61d8efac-26bb-4e5c-9283-ea93bde693b1.png">